packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -497,6 +503,26 @@ export class CallContext {
     });
   }
 
+  public setRemoteVideoStreamViewScalingMode(
```

Comment:
Question:
As Contoso I can do:
```
const view = statefulCallClient.createView()
view.updateScalingMode()
```

when do I call `setRemoteVideoStreamViewScalingMode` over `view.updateScalingMode`?

Reply:
Oh different file and class, never mind!